db/app/model.js

```diff
@@ -43,14 +43,16 @@ exports.selectCommentsByArticleId = (article_id) => {
     )
     .then(({ rows }) => {
       return rows;
-    })
+    });
 };
 
-
 exports.insertCommentByArticleId = (article_id, newComment) => {
   const { username, body } = newComment;
   if (!username || !body) {
-    return Promise.reject({status: 400, msg: "Bad request. Username and Body are required"})
+    return Promise.reject({
+      status: 400,
+      msg: "Bad request. Username and Body are required",
```

Comment:
I would be wary of giving too much info in status messages - it can become tedious to keep track of as a dev making new messages for all the different options and also it can be easy to accidentally give away info about your database.